examples/docs_beta_snippets/docs_beta_snippets/guides/data-modeling/asset-factories/python-asset-factory.py

```diff
@@ -11,7 +13,9 @@ def build_etl_job(
     target_object: str,
     sql: str,
 ) -> dg.Definitions:
-    @dg.asset(name=f"etl_{bucket}_{target_object}")
+    name = f"etl_{bucket}_{target_object}".replace(".", "_")
```

Comment:
I wish we could avoid this but I don't see a great way to do it. I think we should add a comment to the effect of "asset keys cannot contain '.'"